backends/main.py

```diff
@@ -248,10 +268,20 @@ class StartInferenceResponse(BaseModel):
 async def start_text_inference(data: StartInferenceRequest) -> StartInferenceResponse:
     try:
         # Store the current model's configuration for later reference
-        app.text_model_config = data.modelConfig
+        app.state.text_model_config = data.modelConfig
         # Send signal to start server
         model_file_path: str = data.modelConfig["savePath"]
-        isStarted = await start_text_inference_server(model_file_path)
+        app.text_inference_process = (
+            await text_llama_cpp_python.start_text_inference_server(
```

Comment:
This will be removed in favor of our own implementation of inference on homebrew server.